src/miral/simulated_secondary_click.cpp

```diff
@@ -54,6 +55,78 @@ miral::SimulatedSecondaryClick::SimulatedSecondaryClick(bool enabled_by_default)
 {
 }
 
+miral::SimulatedSecondaryClick::SimulatedSecondaryClick(live_config::Store& config_store)
+{
+    config_store.add_bool_attribute(
+        {"simulated_secondary_click", "enable"},
+        "Whether or not to start the compositor with simulated secondary click enabled",
```

Comment:
Technically the user can cause a config update at any time, not just when the compositor starts

```suggestion
        "Whether or not simulated secondary click is enabled",
```